src/media/js/main.js

```diff
@@ -338,5 +338,12 @@ function(_) {
         throw new Error('Cancel navigation; deprecated client');
     });
 
+    // Remove paginated class from app lists if .loadmore goes away.
```

Comment:
Totally agree this was meant to go here just to test the event firing and I forgot about it...was going to place it in search.js but it can have its own module.